### ISSUE TITLE
[Server Request]: Battlefield 3, Battlefield Bad Company 2, Battlefield 2142

### ISSUE THREAD

**Issue by BestNoob** [GameServerManagers/LinuxGSM#4662]
### Game server

Battlefield Dedicated Server

### Linux support

- [X] Yes

### Steam

No

### Guides

https://forums.veniceunleashed.net/viewtopic.php?p=60#p60
https://docs.veniceunleashed.net/hosting/

https://www.moddb.com/mods/bf2142-project-remaster
https://battlefield2142.co/faq/
https://www.moddb.com/games/battlefield-2142/downloads/bf2142-hub-2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by h3o66** [comment]
VU needs wine, we do not support this.
see: https://docs.veniceunleashed.net/hosting/setup-linux/

BF 2142 might be possible, there seem to be serverfiles, but not even sure if it is even automatic possible to install it:
https://www.bf-games.net/downloads/category/326/serverfiles-dedicated-server.html

there is a manual installer, where you need to accept Eula und Punkbuster.

**Comment by dgibbs64** [comment]
I would love to support these games. However, it looks like it might be out of scope for LinuxGSM and although I loved playing 2142 I don't think there is much demand for it anymore.

**Comment by BestNoob** [comment]
Yes ure right, i changed this request to BF2 Battlefield 2 & BF2142 Battlefield 2142 because they are equal and have same base and Setup.

Maybe merge requests:
https://github.com/GameServerManagers/LinuxGSM/issues/2690
